cpg_workflows/stages/mito.py

```diff
@@ -259,3 +262,221 @@ def queue_jobs(
+            + '.crai',
+        )
+        shifted_cram = get_batch().read_input_group(
+            cram=str(inputs.as_path(sequencing_group, RealignMito, 'shifted_cram')),
+            crai=str(inputs.as_path(sequencing_group, RealignMito, 'shifted_cram'))
+            + '.crai',
+        )
+        if get_config()['mito_snv']['use_verifybamid']:
+            verifybamid_output = get_batch().read_input(
+                str(inputs.as_path(sequencing_group, CramQC, 'verify_bamid')),
+            )
+        else:
+            verifybamid_output = None
+
+        # Call variants on WT genome
+        call_j = mito.mito_mutect2(
+            b=get_batch(),
+            cram=non_shifted_cram,
+            reference=mito_ref,
+            region='chrM:576-16024',  # Exclude the control region.
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(call_j)
+        assert isinstance(call_j.output_vcf, hb.ResourceGroup)
+
+        # Call variants in ONLY the control region using the shifted reference
+        shifted_call_j = mito.mito_mutect2(
+            b=get_batch(),
+            cram=shifted_cram,
+            reference=shifted_mito_ref,
+            region='chrM:8025-9144',  # Only call inside the control region.
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(shifted_call_j)
+        assert isinstance(shifted_call_j.output_vcf, hb.ResourceGroup)
+
+        # Merge the wt and shifted VCFs
+        merge_j = mito.liftover_and_combine_vcfs(
+            b=get_batch(),
+            vcf=call_j.output_vcf,
+            shifted_vcf=shifted_call_j.output_vcf,
+            reference=mito_ref,
+            shift_back_chain=shifted_mito_ref.shift_back_chain,
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(merge_j)
+        assert isinstance(merge_j.output_vcf, hb.ResourceGroup)
+
+        # Merge the mutect stats output files (needed for filtering)
+        merge_stats_J = mito.merge_mutect_stats(
+            b=get_batch(),
+            first_stats_file=call_j.output_vcf['vcf.gz.stats'],
+            second_stats_file=shifted_call_j.output_vcf['vcf.gz.stats'],
+            job_attrs=self.get_job_attrs(sequencing_group),
+        )
+        jobs.append(merge_stats_J)
+        assert isinstance(merge_stats_J.combined_stats, hb.ResourceFile)
+
+        # Initial round of filtering to exclude blacklist and high alt alleles
+        initial_filter_j = mito.filter_variants(
+            b=get_batch(),
+            vcf=merge_j.output_vcf,
+            reference=mito_ref,
+            merged_mutect_stats=merge_stats_J.combined_stats,
+            # alt_allele and vaf config hardcoded in this round of filtering as per
+            # https://github.com/broadinstitute/gatk/blob/master/scripts/mitochondria_m2_wdl/AlignAndCall.wdl#L167
+            max_alt_allele_count=4,
```

Comment:
Would we be able to define these hardcoded variables at the top of the page?

Reply:
done.